### ISSUE TITLE
iOS does not support listing WiFi networks (or credentials of current SSID)

### ISSUE THREAD

**Issue by isabelrios** [mozilla-sensorweb/sensorweb-client#6]
Please see screenshot below:
![img_0026](https://cloud.githubusercontent.com/assets/1897507/19469513/ea8f2e46-951b-11e6-8940-fc76971d118a.PNG)

These are the logs shown in xCode for that part:
2016-10-18 10:06:01.853738 SensorWeb[703:356586] THREAD WARNING: ['WifiWizard'] took '25.164062' ms. Plugin should use a background thread.
2016-10-18 10:06:01.909495 SensorWeb[703:356586] WARN: Unable to scan for WiFi networks. Not supported
2016-10-18 10:06:01.909646 SensorWeb[703:356586] current network = undefined ssid ONOA9C7
2016-10-18 10:06:06.934591

**Comment by mcav** [comment]
Yes, this is unfortunately expected for the moment: iOS doesn't allow us to list WiFi networks, nor does it allow us to know whether or not the current network requires a password.

I've updated the title to encompass both cases, as this bug will be around for a while until we decide whether or not the sensor is going to send the list of WiFi networks to the phone via bluetooth.
